page_creation.rst

```diff
@@ -65,44 +60,154 @@ random) number and prints it. To do that, create a "Controller class" and a
         }
     }
 
-Before diving into this, test it out! If you are using PHP's internal web server
-go to:
+To map a URL to this controller, create a route in ``config/routes.yaml``:
```

Comment:
I just wonder if this is too advanced to be understood by someone who is starting with Symfony:

```
To map a URL to this controller, create a route in ``config/routes.yaml``:
```

Would something like this be better ... or just more verbose and complicated?

```
Now you need to associate this controller with a public URL so the ``number()``
method is executed when users browse some URL (e.g. ``/lucky/number``). This
association is defined creating a **route** in the ``config/routes.yaml`` file:
```